src/Mono.Android/metadata

```diff
@@ -1528,4 +1528,27 @@
   <attr api-since="29" path="/api/package[@name='java.lang']/class[@name='StringBuilder']" name="extends-generic-aware">java.lang.AbstractStringBuilder</attr>
   <!-- We don't need to remove any method from Java.Lang.AbstractStringBuilder because they are all virtual methods -->
 
+  <!-- Api 30 adjustments -->
+  <!-- We need to remove abstract method close from CameraOfflineSession since it is already defined on base abstract class CameraCaptureSession -->
+  <remove-node api-since="30" path="/api/package[@name='android.hardware.camera2']/class[@name='CameraOfflineSession']/method[@name='close' and count(parameter)=0]" />
+
+  <!-- Namespace android.view.inline should be renamed to Android.Views.Inline -->
+  <attr api-since="30" path="/api/package[@name='android.view.inline']" name="managedName">Android.Views.Inline</attr>
+
+  <!-- Google has added getCellIdentity and getCellSignalStrength as abstract method on the base class -->
+  <attr api-since="30" path="/api/package[@name='android.telephony']/class[@name='CellInfoCdma']/method[@name='getCellIdentity' and count(parameter)=0]" name="return">android.telephony.CellIdentity</attr>
```

Comment:
Why do we need to override the `return` attribute?  That feels dubious/problematic.

Reply:
```zsh
% javap -s -classpath ~/android-toolchain/sdk/platforms/android-29/android.jar android.telephony.CellInfoCdma
Compiled from "CellInfoCdma.java"
public final class android.telephony.CellInfoCdma extends android.telephony.CellInfo implements android.os.Parcelable {
  public android.telephony.CellIdentityCdma getCellIdentity();
    descriptor: ()Landroid/telephony/CellIdentityCdma;
  ...
}
```

The return signature is `Landroid/telephony/CellIdentityCdma;`.  This change should *prevent* the member from being usable, as `JNIEnv.GetMethodID(..., "getCellIdentity", "()Landroid/telephony/CellIdentity;")` should return NULL.